app/views/restaurants/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<div style="text-align:center;"> 
```

Comment:
Do we need this file?

Reply:
yes, i am showing admin interface here, there wiil be a button 'Add Restaurant', so it will contain multiple card of restaurants